arctic/serialization/numpy_records.py

```diff
@@ -12,11 +13,20 @@
     except ImportError:  # <= 0.19 Compatibility
         from pandas.tslib import Timestamp, get_timezone
 
+from ..exceptions import ArcticException
 
 log = logging.getLogger(__name__)
 
 DTN64_DTYPE = 'datetime64[ns]'
 
+# TODO: Switch on by default this flag to enable the fast check once this gets thoroughly tested
+_FAST_CHECK_DF_SERIALIZABLE = 'ENABLE_FAST_CHECK_DF_SERIALIZABLE' in os.environ
```

Comment:
What if someone sets ENABLE_FAST_CHECK_DF_SERIALIZABLE=0?

Reply:
yeah, this is a better convention instead: 
```python
"bool(os.environ.get('ENABLE_FAST_CHECK_DF_SERIALIZABLE'))"  
```